pegasus/sites.v3/code.org/public/curriculum/csp/docs/postap.md

```diff
@@ -9,7 +9,7 @@ video_player: true
 
 The following table is intended to outline the main programming topics for the post-AP material in Code Studio, so that teachers may familiarize themselves with the material and feel more comfortable assisting students.
 
-In Code Studio, there is a <a href="https://studio.code.org/s/cspunit6/" target="_blank">collection of levels</a> that are intended to familiarize students with the different components of creating data-driven apps in App Lab. These stages will take several (approximately 4) weeks to complete, and students should work through the material at their own pace. Stage 8 includes examples of apps that can be made using the data tools, which students can use as inspiration for their final projects. The final stage contains a single level, in which students may build their final projects.
+In Code Studio, there is a <a href="https://studio.code.org/s/cspunit6?viewAs=Instructor" target="_blank">collection of levels</a> that are intended to familiarize students with the different components of creating data-driven apps in App Lab. These stages will take several (approximately 4) weeks to complete, and students should work through the material at their own pace. Stage 8 includes examples of apps that can be made using the data tools, which students can use as inspiration for their final projects. The final stage contains a single level, in which students may build their final projects.

```

Comment:
Not the job of this PR but looking at this page. This page is so old. We should remove it.